contents/docs/api/post-only-endpoints.mdx

```diff
@@ -99,6 +99,39 @@ curl -v -L --header "Content-Type: application/json" -d '{
 }' https://app.posthog.com/capture/
 ```
 
+### Group
+
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "event": "$event",
+    "distinct_id": "1234",
+    "properties": {
+        "$groups": {"company": "<company_name>"}
+    }
+}' https://app.posthog.com/capture/
+```
+
+
+### Group identify
+
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "event": "$groupidentify",
+    "properties": {
+        "distinct_id": "company_<company_name>",
+        "$group_type": "<group_type>",
+        "$group_key": "<company_name>",
+        "$group_set": {
+            "name": "<company_name>",
+            "user_count": 5,
```

Comment:
let's maybe use something that's not going to change ... because when someone does analytics we use values from the time of the event and this could lead folks to think they can later filter on the updated user count, which they can't with PoE, so maybe sth like `subscription: premium` would be good here as for that you would want at the time of the event values.

Reply:
might want to change it https://posthog.com/docs/product-analytics/group-analytics too for all of them